tests/metrics/test_base.py

```diff
@@ -14,18 +14,30 @@ def test_evaluation_example_str():
         )
     )
     example1_expected = """
-        Input: This is an input
-        Provided output: This is an output
-        Provided foo: bar
-        Score: 5
-        Justification: This is a justification
+        Input:
+        This is an input
+
+        Output:
+        This is an output
+
+        Additional information used by the model:
+        key: foo
+        value:
+        bar
+
+        score: 5
+        justification: This is a justification
         """
     assert re.sub(r"\s+", "", example1_expected) == re.sub(r"\s+", "", example1)
 
     example2 = str(EvaluationExample(input="This is an input", output="This is an output", score=5))
     example2_expected = """
-        Input: This is an input
-        Provided output: This is an output
-        Score: 5
+        Input:
+        This is an input
+
+        Output:
+        This is an output
+
+        score: 5
```

Comment:
Justification seems to be missing here, but we tell the LLM that they need to produce a justification in the prompt. Shall we require justification?